### ISSUE TITLE
Determine optimal build flags for primecount C library

### ISSUE THREAD

**Issue by pgujjula** [pgujjula/primecount-haskell#3]
depends on #2

**Comment by pgujjula** [comment]
Building `libprimecount` from source does have a speedup, probably because POPCNT is enabled.

**Comment by pgujjula** [comment]
No appreciable difference between using `-fllvm` and normal GHC backend.

**Comment by pgujjula** [comment]
Using `clang++` instead of `g++` has a miniscule effect, if anything, on the performance of `libprimecount` (5.9 secs vs 6.0 secs to compute `primePi (10^16)` with a single-thread.)

**Comment by pgujjula** [comment]
To conclude, seems like the only thing affecting performance is whether or not POPCNT is enabled, which is already covered by the original documentation for `primecount`. So it seems like we don't have to do anything extra here.
